src/ontology/cl-edit.owl

```diff
@@ -26609,7 +26613,10 @@ SubClassOf(obo:CL_1000697 obo:CL_1000500)
 
 # Class: obo:CL_1000698 (kidney resident macrophage)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_1000698 "A macrophage that resides in the kidney.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_1000698 "KUPO:0001109"^^xsd:string)
+AnnotationAssertion(oboInOwl:reference obo:CL_1000698 "PMID:36066976")
+AnnotationAssertion(oboInOwl:reference obo:CL_1000698 "PMID:6854206")
```

Comment:
It looks like this PR is not passing QC checks because of these reference annotations.

'database_cross_reference' is used as the annotation property for referencing in CL.
If these references support the text definition, please add them to the text definition using 'database_cross_reference'.